### PR TITLE
Lesson 2

### PR DESCRIPTION
**Pull request by oiskritsky**
discription

### CODE REVIEW

File: about/index.html [hunk 1]
```diff
@@ -1,12 +1,30 @@
 <!DOCTYPE html>
```
Comment (ghost) [inline about/index.html]: Make your HTML files prettier. Remove extra spaces and write each tag with a new line.
```suggestion
<p>
     <span>Some text<span>
</p>
```

File: about/index.html [hunk 2]
```diff
@@ -1,12 +1,30 @@
...
+            <h3><i>Заголовок 3-го уровня</i></h3>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+                <h5>Заголовок 5-го уровня</h5>
+                        <p><pre>
+                                &nbsp&nbspFusce porttitor lorem id mi iaculis feugiat.
+                                Etiam non mauris non diam vulputate mollis.
+                                Pellentesque habitant morbi tristique senectus et netus
+                                et malesuada fames ac turpis egestas.</pre></p>
+            <!-- <br> -->
+            <hr>
```
Comment (ghost) [inline about/index.html]: Make a closing tag.

File: about/index.html [hunk 3]
```diff
@@ -1,12 +1,30 @@
...
+        <blockquote>
+            <p><strong>...Даже если змея мечтает летать на небе, всё равно ей суждено ползать на земле.</strong></p>
+        </blockquote>
+            <hr>
```
Comment (ghost) [inline about/index.html]: Make a closing tag.

File: about/index.html [hunk 4]
```diff
@@ -1,12 +1,30 @@
...
+        <p><em><a href="https://github.com/oiskritsky">Я на GitHub</a></em></p>  
+            <hr>
```
Comment (ghost) [inline about/index.html]: Make a closing tag.

File: about/index.html [hunk 5]
```diff
@@ -1,12 +1,35 @@
...
-    <title>Document</title>
-</head>
-<body>
-    
-</body>
+<html lang="ru">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (ghost) [inline about/index.html]: All meta tags should be closed. <meta />

File: index.html [hunk 6]
```diff
@@ -1,12 +1,17 @@
-<!DOCTYPE html>
-<html lang="en">
+<html lang="ru">
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (ghost) [inline index.html]: Same problem with closing tags.

Comment (oiskritsky) [reply index.html]: done

File: about/index.html (hunk 5 above)
Comment (oiskritsky) [reply about/index.html]: i've done

**Pull request merged**